packages/esm-patient-tests-app/src/lab-orders/api.ts

```diff
@@ -81,6 +81,8 @@ export function prepLabOrderPostData(
   patientUuid: string,
   encounterUuid: string | null,
 ): OrderPost {
+  const urgency = order.urgency ?? 'ROUTINE';
```

Comment:
```suggestion
```

`urgency` is a required field in the form, so we shouldn't need to assign a default value.